trade-finance-manager-ui/server/constants/bank-reports-for-period-table-header-prefix.ts

```diff
@@ -9,8 +9,6 @@ const REPORTING_PERIOD_METHOD_OF_DESCRIPTION = {
 } as const;
 
 export const BANK_REPORTS_FOR_PERIOD_TABLE_HEADER_PREFIX = {
-  CURRENT_PERIOD: `${REPORT_PERIOD_DISCRIPTORS.CURRENT} ${REPORTING_PERIOD_METHOD_OF_DESCRIPTION.BY_FULL_PERIOD}`,
```

Comment:
Please can we also remove the 'BY_FULL_PERIOD' above as well and can then probably flatten the const it is in to something just like `const REPORTING_PERIOD = 'reporting period'`

Reply:
Could possibly also simplify the namings of these to just 'CURRENT_PERIOD' and 'OPEN_PERIOD' since no need to distinguish